firmware/tunerstudio/rusefi.input

```diff
@@ -1238,7 +1238,7 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 		panel = ignitionCylExtra,	East
 		
 ;			Engine->Aux1 Thermistor Settings
-	dialog = auxTempSensor1_thermistor, "aux1 Thermistor Settings"
+	dialog = auxTempSensor1Sensor,	"aux1 Thermistor Settings"
```

Comment:
Renaming this to change panel structure a little bit: we removed auxTempSensor1Sensor_IO panel below, so auxTempSensor1_thermistor is the only one left in auxTempSensor1Sensor